src/global/helpers/host-context.ts

```diff
@@ -18,3 +18,5 @@ export function hostContext(selector: string, element: Element): Element | null
 
   return null;
 }
+
+export const ACTION_ELEMENTS = 'a,button,sbb-card,sbb-teaser-hero,sbb-teaser';
```

Comment:
Maybe add documentation?

Reply:
A list of elements that should not allow another anchor or button element inside them.